engine-rest/engine-rest/src/test/java/org/camunda/bpm/engine/rest/ProcessInstanceRestServiceInteractionTest.java

```diff
@@ -3283,17 +3339,21 @@ public void testSetRetriesByProcessAsyncWithQuery() {
 
     verifyBatchJson(response.asString());
 
-    verify(mockManagementService, times(1)).setJobRetriesAsync(
-        eq((List<String>) null), Mockito.<ProcessInstanceQuery>any(), Mockito.eq(5));
+    verify(mockManagementService, times(1)).setJobRetriesByProcessAsync(eq(MockProvider.EXAMPLE_JOB_RETRIES));
+    verify(mockSetJobRetriesByProcessAsyncBuilder, times(1)).processInstanceIds(null);
+    verify(mockSetJobRetriesByProcessAsyncBuilder, times(1)).processInstanceQuery(null);
```

Comment:
Same question regarding the `processInstanceQuery` as above, shouldn't this have been called with a value?